src/main/java/pl/kl/weatherservice/WeatherServiceApplication.java

```diff
@@ -1,13 +1,30 @@
 package pl.kl.weatherservice;
 
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import pl.kl.weatherservice.security.User;
+import pl.kl.weatherservice.security.UserRepository;
+
+import java.util.Collections;
 
 @SpringBootApplication
-public class WeatherServiceApplication {
+@RequiredArgsConstructor
+public class WeatherServiceApplication implements CommandLineRunner {
+
+    private final UserRepository userRepository;
+    private final PasswordEncoder passwordEncoder;
 
     public static void main(String[] args) {
         SpringApplication.run(WeatherServiceApplication.class, args);
     }
 
+    @Override
+    public void run(String... args) throws Exception {
```

Comment:
let's try to move it to a new class and create it as spring component with specific profile like 'develop'